agent/version/version.go

```diff
@@ -22,10 +22,10 @@ package version
 // repository. Only the 'Version' const should change in checked-in source code
 
 // Version is the version of the Agent
-const Version = "1.61.0"
+const Version = "1.61.1"
 
 // GitDirty indicates the cleanliness of the git repo when this agent was built
 const GitDirty = true
 
 // GitShortHash is the short hash of this agent build
-const GitShortHash = "b00e06d9"
+const GitShortHash = "5ca80892"
```

Comment:
Should this be be393c83e8938273616ad706108cf7b78cfc8374 for the latest PR https://github.com/aws/amazon-ecs-agent/pull/3197?

Reply:
good catch, I re-ran ```go run gen/version-gen.go``` after rebasing.